tests/contrib/anthropic/test_anthropic_llmobs.py

```diff
@@ -494,7 +513,18 @@ def test_tools_sync_stream(self, anthropic, ddtrace_global_config, mock_llmobs_w
                 input_messages=[{"content": WEATHER_PROMPT, "role": "user"}],
                 output_messages=[
                     {"content": message[0]["text"], "role": "assistant"},
-                    {"content": message[1]["text"], "role": "assistant"},
+                    {
+                        "content": "",
+                        "role": "assistant",
+                        "tool_calls": [
+                            {
+                                "name": "get_weather",
+                                "arguments": {"location": "San Francisco, CA"},
+                                "tool_id": "",
```

Comment:
No tool ID here?

Reply:
Offline discussion - looks like we are not correctly extracting tool iD from streamed tool call responses. Will address this in a separate PR